packages/flex-plugin-scripts/src/utils/__tests__/run.test.ts

```diff
@@ -69,4 +69,35 @@ describe('run', () => {
     expect(print).toHaveBeenCalledTimes(1);
     expect(details).toHaveBeenCalledTimes(1);
   });
+
+  describe('exit', () => {
+    it('should quit if isRequiredScript is true', () => {
```

Comment:
`should quit if isRequiredScript is false`?

Reply:
yep will fix once you finish the review